### ISSUE TITLE
To Do's ideas

### ISSUE THREAD

**Issue by fskinner** [Cmdv/cycle-webpack-boilerplate#21]
Sorry, i didn't know where to put these... So i've made this issue

What do you think about adding two more pages, so new people can have more examples to learn from?

I was thinking in there two examples:
- **Random Github User (or anything like that)**: this example would use HTTPDriver. That's nothing really difficult here, the trick would be adding the new source into main without breaking the other existing pages

- **Simple Todo app**: this example would show more streams that affects the final state like typing, adding a todo, removing a todo and would also generate a more complex view (because you would need a remove button for each todo for example)

I guess the todo app would be more for helping beginners learn rx/cycle... But the first one could be used to show how to add a driver with best practices

What do you think? :)

**Comment by Cmdv** [comment]
Hi @fskinner sorry for the late reply this went under my radar for some reason. But yeah that sounds like a cool idea for sure so different pages would show different user cases.

We have to be a little careful that we don't pack the boilerplate with too much information but it does lack HTTP so that would be a an ace thing to introduce.

In the long run I was thinking of converting parts of this repo into a build, where users can chose what options they want the boilerplate to have. Webpack / Browserify, HTTP / Websockets etc so yes more example will definitely help toward building that.

Did you want to build them or were you looking for them to be built? 😄

**Comment by fskinner** [comment]
That's okay :)

I have a HTTP example but I don't know if its wired correctly to the rest of cyclejs since I've done `const Http = Content.HTTP || {};` on the main() function... Couldn't come up with a better way of setting HTTPDriver without the need of receiving/returning it on the other examples (home/page1/page2).

I do not have much experience with RxJS so I don't know if I'm doing something really weird :smile:
I could make a PR with the random user/quote using HTTP and the simple Todo list so you can review it and decide if its worth to add into the boilerplate.

**Comment by fskinner** [comment]
On the other hand I agree with you about not packing it with too much information as it would become some sort of a tutorial instead of a boilerplate/starter

**Comment by Cmdv** [comment]
yeah thats cool, why not concentrate on the HTTP and will review it I'm sure you are close 👍
